app/models/foreman_openscap/policy.rb

```diff
@@ -219,15 +245,30 @@ def ensure_needed_puppetclasses
     end
 
     def cron_line_split
-      cron_line.split(' ')
+      cron_line.to_s.split(' ')
     end
 
     def valid_cron_line
-      return true if period != 'custom' || step_index != 4
+      if period == 'custom' && should_validate?(4)
+        errors.add(:cron_line, _("does not consist of 5 parts separated by space")) unless cron_line_split.size == 5
+      else
+        errors.add(:cron_line, _("should be empty")) unless cron_line.blank?
```

Comment:
I think this could be implemented by standard `validate :cron_line, :presence => true, :if => :cron_line.present?`